.github/scripts/lint-ecosystem.js

```diff
@@ -52,19 +51,48 @@ module.exports = async function ({ core }) {
     modules.push(moduleName)
   }
 
+  const failures = []
   if (hasOutOfOrderItem === true) {
-    core.setFailed('Some ecosystem modules are not in alphabetical order.')
+    const failure = 'Some ecosystem modules are not in alphabetical order.'
+    failures.push(failure)
+    core.setFailed(failure)
   }
 
   if (hasImproperFormat === true) {
-    core.setFailed('Some ecosystem modules are improperly formatted.')
+    const failure = 'Some ecosystem modules are improperly formatted.'
+    failures.push(failure)
+    core.setFailed(failure)
+  }
+
+  handleFailures(failures, { core, github, context })
+}
+
+function handleFailures (failures, workflow) {
+  const { github, context, core } = workflow;
+
+  if (failures.length) {
+
+    const output = `
+    ## :x: Failed: Lint Ecosystem Order
+
+    ${failures.map((failure) => '- ' + failure + '\n')} 
+
+    [View Details](https://github.com/fastify/fastify/actions/runs/${context.runId}/jobs/${context.job})
+    `
+
+    github.rest.issues.createComment({
```

Comment:
Should we await it?

Reply:
I think it won't work for security reasons:

> Unhandled error: HttpError: Resource not accessible by integration

I must create a branch in this repository

Reply:
Yeah, It works but the message must be fixed a bit:
https://github.com/fastify/fastify/pull/4279#issuecomment-1248337226